test/prettyprinter/dune

```diff
@@ -0,0 +1,3 @@
+(test
+ (name prettyprinter)
```

Comment:
let's go for `pretty-printer` as the library name and the directory name here

Reply:
Looks like there is an issue with the name with a hyphen:
```
File "test/pretty-printer/pretty-printer.pp.ml", line 1:
Error (warning 24): bad source file name: "Dune__exe__Pretty-printer" is not a valid module name.
```

Maybe we should use `pretty_printer` instead of `pretty-printer`? Or disabling this warning is preferred?

Reply:
Yep, hypens are not allowed in module names, it was just meant for the library. We can just use `Test.ml` for the module, i.e. `test/pretty-printer/Test.ml`  (or `ppTest.ml`).